### DISCUSSION TITLE
Curated list of plugins?

### DISCUSSION THREAD

**Discussion by armenr** [vuejs/pinia#577] [Ideas]
Is there any info or a curated list of pinia plugins that are out there, in the wild? :)

Thanks for all your hard work on this!

**Comment by posva** [comment]
No but at this point just searching on npm or GitHub will pretty much give you anything that exists

**Comment by posva** [comment]
At the moment, this works pretty well: https://www.npmjs.com/search?q=pinia

**Reply by sschneider-ihre-pvs** [reply]
https://www.npmjs.com/search?q=keywords:pinia-plugin guess this narrows it down a bit more
